tests/languages/tt2/keyword_feature.test

```diff
@@ -0,0 +1,161 @@
+[% GET foo %]
+[% CALL dbi.disconnect %]
+[% SET answer = 42 %]
+[% DEFAULT answer = 42 %]
+[% INSERT filename.html %]
+[% INCLUDE "$inc" %]
+[% PROCESS "functions.tt" %]
+[% WRAPPER 'html5.html' %]
+[% BLOCK header %]
+[% RETURN %]
+[% END %]
+[% IF foo %]
+[% ELSIF bar %]
+[% ELSE %]
+[% END %]
+[% UNLESS fear %]
+[% END %]
+[% SWITCH mday %]
+[% CASE 23 %]
+[% CASE %]
+[% END %]
+[% FOREACH project IN time %]
+[% LAST IF exhausted %]
+[% NEXT IF client %]
+[% END %]
+[% WHILE my.guitar.gently.weeps %]
+[% END %]
+[% FILTER html %]
+[% END %]
+[% USE Common::Sense %]
+[% END %]
+[% MACRO header INCLUDE header %]
+[% RAWPERL %]
+[% END %]
+[% PERL %]
+[% END %]
+[% TRY %]
+[% THROW up %]
+[% CATCH the.ball %]
+[% FINAL breathe %]
+[% END %]
+[% STOP %]
+[% CLEAR %]
+[% META title = 'Hello!' %]
+[% TAGS [@ @] %]
+[% DEBUG on %]
+[% DEBUG off %]

```

Comment:
I understood that but I found a bug or two because of the more realistic test cases.  Is that really a problem? It's better test coverage in the end.

Reordering: see above, whatever you prefer.  Although `ELSE` before `IF` looks really strange to me. ;)

Reply:
No, it's not really a problem. It was just a suggestion, feel free to let it as it is.